xchange-bitfinex/src/main/java/org/knowm/xchange/bitfinex/v2/dto/marketdata/BitfinexTickerFundingCurrency.java

```diff
@@ -2,12 +2,14 @@
 
 import com.fasterxml.jackson.annotation.JsonFormat;
 import java.math.BigDecimal;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
 import lombok.AllArgsConstructor;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
 import lombok.ToString;
 
 @JsonFormat(shape = JsonFormat.Shape.ARRAY)
+@JsonPropertyOrder({"symbol", "frr", "bid", "bidPeriod", "bidSize", "ask", "askPeriod", "askSize", "dailyChange", "dailyChangePerc", "lastPrice", "volume", "high", "low", "placeHolder0", "placeHolder1", "frrAmountAvailable"})
```

Comment:
this is just the natural order of properties

I don't see why this would be needed

Reply:
I added this to fix this issue - https://github.com/knowm/XChange/issues/3246 which i was seeing also

Reply:
#3246 relates to a different DTO

please add a unit test first that show that we are unable to parse these DTOs without `@JsonPropertyOrder` annotations because to my understanding on how Jackson works your code did not change anything

Reply:
The issue only happens on Android, i cant seem to reproduce now in unit tests although im sure i could yesterday. Do you think there is anything wrong with explicitly setting the order? if so no problem i will just use my fork

Reply:
if its not needed then it should not be there - just that :)

if you don't add a unit test that tests it then one day someone will add a field here and forget to add it in the annotation

but that said we are lacking a CI job that would run tests on the Androids JVM in this project - something to consider